pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -44,36 +42,44 @@ var (
 		"packetDeltaCount",
 		"octetDeltaCount",
 	}
+	IANAInfoElementsIPv4 = append(IANAInfoElementsCommon, []string{"sourceIPv4Address", "destinationIPv4Address"}...)
+	IANAInfoElementsIPv6 = append(IANAInfoElementsCommon, []string{"sourceIPv6Address", "destinationIPv6Address"}...)
+	// Substring "reverse" is an indication to get reverse element of go-ipfix library.
 	IANAReverseInfoElements = []string{
 		"reversePacketTotalCount",
 		"reverseOctetTotalCount",
 		"reversePacketDeltaCount",
 		"reverseOctetDeltaCount",
 	}
-	AntreaInfoElements = []string{
+	AntreaInfoElementsCommon = []string{
```

Comment:
does not need to be exported I think... could be `antreaInfoElementsCommon`

Reply:
Indeed. Updated.